src/sentry/static/sentry/app/components/installPromptBanner.jsx

```diff
@@ -41,11 +40,11 @@ const InstallPromptBanner = createReactClass({
   },
 
   sentFirstEvent() {
-    let {onboardingTasks} = this.props.organization;
-    let firstEventTask = onboardingTasks.find(task => task.task === 2);
-    this.setState({
-      sentFirstEvent: firstEventTask && firstEventTask.status === 'complete',
-    });
+    let {projects} = this.props.organization;
+    let hasFirstEvent = projects.find(
+      project => project.firstEvent && project.firstEvent !== null
```

Comment:
Is the null comparison necessary, anything truthy will never be equal to `null`

```suggestion
      project => project.firstEvent
```